website/docs/r/codespaces_organization_secret_repositories.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Can you link to this doc from the `github.erb` file so it's discoverable?

Reply:
Added, thank you for the review!